pkg/controller/pattern1/deployment.go

```diff
@@ -522,4 +539,29 @@ func WorkerDeployment(apimanager *apimv1alpha1.APIManager, y *configvalues, num
 			},
 		},
 	}
+
+}
+
+// AssignSecurityContext is ..
+func AssignSecurityContext(securityContextString []string, securityContext *corev1.SecurityContext) {
+
+	if securityContextString[0] == "runAsUser" {
```

Comment:
It would have been much clearer if you could refactor this with a switch case. WDYT? Also, if there context strings are used in other places, let's define constants and use them here.

Reply:
fixed via https://github.com/BLasan/k8s-wso2am-operator/commit/58af6aead63163f353b22d0645fda8165d80d3e1